docs/howtos/layers/vectors.md

```diff
@@ -78,54 +130,38 @@ viewer.close()
 accept the same layer-creation parameters.
 
 ```{code-cell} python
-:tags: [hide-cell]
+:tags: [hide-output]
 
 help(napari.view_vectors)
 ```
 
```

Comment:
Same comment as previously.

Reply:
Peter, you are just the best.  Thank you for catching these errors!